packages/server/src/provisioning/stripe.ts

```diff
@@ -90,6 +90,8 @@ export class StripeProvisioner extends BasicProvisioner {
                 "Shared vaults",
                 "Encrypted file storage",
                 "Security Report",
+                "Built-in Authenticator",
+                "Advanced Note-Taking",
```

Comment:
If you have a better idea on how to name these features, let me know. For the notes, I've also considered:
- Richtext / Markdown Notes
- Full-featured Notes Editor

From what I've seen elsewhere "Authenticator" is commonly understood to mean "Google Authenticator / Authy type functionality". "Multi-Factor Authentication" is ambiguous because it can also mean MFA for logging into Padloc.